template/README.md

```diff
@@ -0,0 +1,19 @@
+This readme should give people everything they need to maintain the scraper.
+
+# Summary
+Is there anything in particular of note with the scraping landscape in this jurisdiction?
+Time period of data (e.g. 06/01/2017 to 03/20/2021)
```

Comment:
```suggestion
Time period of data (e.g. 06/01/2017 to 03/20/2021)

_Remove fields that were collected_
Fields that could not be obtained within the PDAP legal guidelines:
* _id
* _state
* _county
* CaseNum
* FirstName
* MiddleName
* LastName
* Suffix
* DOB
* Race
* Sex
* ArrestDate
* FilingDate
* OffenseDate
* DivisionName
* CaseStatus
* DefenseAttorney
* PublicDefender
* Judge
* ChargeCount
* ChargeStatute
* ChargeDescription
* ChargeDisposition
* ChargeDispositionDate
* ChargeOffenseDate
* ChargeCitationNum
* ChargePlea
* ChargePleaDate
* ArrestingOfficer
* ArrestingOfficerBadgeNumber

```

Reply:
I accidentally messed up this suggestion in my first review. Hopefully this makes sense.